lib/common/options.c

```diff
@@ -640,12 +639,11 @@ pcmk__format_option_metadata(const char *name, const char *desc_short,
 {
     /* big enough to hold "pacemaker-schedulerd metadata" output */
     GString *s = g_string_sized_new(13000);
-    int lpc = 0;
 
-    g_string_append_printf(s, "<?xml version=\"1.0\"?>"
-                              "<resource-agent name=\"%s\">\n"
-                              "  <version>%s</version>\n",
-                              name, PCMK_OCF_VERSION);
+    pcmk__g_strcat(s,
+                   "<?xml version=\"1.0\"?>"
+                   "<resource-agent name=\"", name, "\">\n"
```

Comment:
It would make sense to add version= with the Pacemaker version

Reply:
Looks like this function is only called to build the metadata string for Pacemaker daemons (correct me if I'm wrong), so yeah that would make sense.